install/installer/pkg/postprocess/postprocess.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package postprocess
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	openvsxproxy "github.com/gitpod-io/gitpod/installer/pkg/components/openvsx-proxy"
+	"github.com/mikefarah/yq/v4/pkg/yqlib"
+	logging "gopkg.in/op/go-logging.v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+// Processors list of processes executed on each resource document
+var Processors = []Processor{
+	// Remove "status" from root of all network policies
+	{
+		Type:       common.TypeMetaNetworkPolicy,
+		Expression: "del(.status)",
+	},
+	// Remove "status" from root of OpenVSXProxy stateful sets
+	{
+		Type:       common.TypeMetaStatefulSet,
+		Expression: "del(.status)",
+		Name:       pointer.String(openvsxproxy.Component),
+	},
+}
+
+type Processor struct {
+	Type       metav1.TypeMeta
+	Expression string
+	Name       *string // Optional
+}
+
+// process emulates how YQ parsers the file
+func process(expression string, obj *common.RuntimeObject) error {
+	// Stop the logging to Stderr
+	var format = logging.MustStringFormatter(
+		`%{color}%{time:15:04:05} %{shortfunc} [%{level:.4s}]%{color:reset} %{message}`,
+	)
+	var backend = logging.AddModuleLevel(
+		logging.NewBackendFormatter(logging.NewLogBackend(os.Stderr, "", 0), format))
+
+	backend.SetLevel(logging.ERROR, "")
+	logging.SetBackend(backend)
+	// End of logger config
+
+	yqlib.InitExpressionParser()
+
+	var writer bytes.Buffer
+	printerWriter := yqlib.NewSinglePrinterWriter(&writer)
+	encoder := yqlib.NewYamlEncoder(2, false, false, true)
+
+	printer := yqlib.NewPrinter(encoder, printerWriter)
+
+	decoder := yqlib.NewYamlDecoder()
+
+	streamEvaluator := yqlib.NewStreamEvaluator()
+
+	dir, err := os.MkdirTemp("", "post-processor")
```

Comment:
Any specific reason, on writing into a local FS and using that, instead of directly creating a reader and using `Evaluate`? 🤔 https://github.com/mikefarah/yq/blob/v4.26.1/pkg/yqlib/stream_evaluator.go#L92

Reply:
I originally tried that but I couldn't get it to work. We already use the "[create temp file, amend, move on](https://github.com/gitpod-io/gitpod/blob/main/install/installer/pkg/helm/helm.go#L102,L114)" approach in Helm so took view that it was ok to use it again

Reply:
/hold actually, just had a brainwave...

Reply:
@Pothulapati Thanks for questioning that. Believe it or not, your question actually made me realise the problem and the solution. I've now changed to using the `Evaluate` method 👍🏻 

/unhold